### PR TITLE
Update debugging with VS code instructions

### PR DESCRIPTION
**Pull request by wtgodbe**
For https://github.com/dotnet/corefx/issues/21557. CC @tmds

### CODE REVIEW

File: Documentation/debugging/unix-instructions.md [hunk 1]
```diff
@@ -65,13 +65,16 @@ lldb-3.9 -O "settings set target.exec-search-paths /home/parallels/Downloads/Sys
...
 - Open the debug window: `ctrl-shift-D` or click on the button on the left
 - Click the gear button at the top to create a launch configuration, select `.NET Core` from the selection dropdown
 - In the `.NET Core Launch (console)` configuration do the following
   - delete the `preLaunchTask` property
-  - set `program` to the full path to corerun in the test directory
-  - set `cwd` to the test directory
+  - set `program` to the full path to `dotnet` in the bin directory.
+    - something like `corefx/bin/testhost/netcoreapp-Linux-{Configuration}-{Architecture}`, plus the full path to your corefx directory.
+  - set `cwd` to the test bin directory.
+    - using the System.Net.Sockets example, should be something like `corefx/bin/tests/System.Net.Sockets.Tests/netcoreapp-Linux-{Configuration}-{Architecture}`, plus the full path to your corefx directory.
```
Comment (stephentoub) [inline Documentation/debugging/unix-instructions.md]: Nit: it should be

Comment (wtgodbe) [reply Documentation/debugging/unix-instructions.md]: It should be what?

Comment (stephentoub) [reply Documentation/debugging/unix-instructions.md]: "using the System.Net.Sockets example, should be something" =>
"using the System.Net.Sockets example, it should be something"

Comment (wtgodbe) [reply Documentation/debugging/unix-instructions.md]: Oh, gotcha. Will change

**Pull request merged**